src/plotrecipes.jl

```diff
@@ -0,0 +1,105 @@
+shapecoords(geom::AbstractPoint) = [tuple(coordinates(geom)...)]
+RecipesBase.@recipe f(geom::AbstractPoint) = (seriestype --> :scatter; legend --> :false; shapecoords(geom))
+
+shapecoords(geom::AbstractVector{<:AbstractPoint}) = Tuple{Float64,Float64}[tuple(coordinates(g)...) for g in geom]
+
+function shapecoords(geom::AbstractMultiPoint)
+    coords = coordinates(geom)
+    first.(coords), last.(coords)
+end
+RecipesBase.@recipe f(geom::AbstractMultiPoint) = (seriestype --> :scatter; legend --> :false; shapecoords(geom))
+
+function shapecoords(geom::Vector{<:AbstractMultiPoint})
+    x = Float64[]; y = Float64[]
+    for g in geom
+        coords = coordinates(g)
+        append!(x, first.(coords)); append!(y, last.(coords))
+    end
+    x, y
+end
+
+function shapecoords(geom::AbstractLineString)
+    coords = coordinates(geom)
+    first.(coords), last.(coords)
+end
+RecipesBase.@recipe f(geom::AbstractLineString) = (seriestype --> :line; legend --> :false; shapecoords(geom))
+
+function shapecoords(geom::Vector{<:AbstractLineString})
+    x = Vector{Float64}[]; y = Vector{Float64}[]
```

Comment:
Should the brackets be parenthesis here or a vector of vector is intended?

Reply:
a vector of vectors

Reply:
Awesome, thanks for double checking.